index.html

```diff
@@ -0,0 +1,195 @@
+              standard dummy text.
+            </p>
+            <ul class="prog-langs">
+              <li>css</li>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <button type="button" class="btn">See Project</button>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="recent-work-container1">
+      <div class="my-works">
+        <div class="my-work-contain">
+          <div class="my-work-card">
+            <img src="./images/ImgPlaceholder.png" alt="image" />
+            <h3 class="title">Multy-Post Stories</h3>
+            <p class="work-description">
+              A daily selection of privately personalized reads; no accounts
+              or sign-ups required. has been the industry's standard dummy
+              text ever since the 1500s, when an unknown printer took a
+              standard dummy text.
+            </p>
+            <ul class="prog-langs">
+              <li>css</li>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <button type="button" class="btn">See Project</button>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="recent-work-container2">
+      <div class="my-works">
+        <div class="my-work-contain">
+          <div class="my-work-card">
+            <img src="./images/ImgPlaceholder.png" alt="image" />
+            <h3 class="title">Multy-Post Stories</h3>
+            <p class="work-description">
+              A daily selection of privately personalized reads; no accounts
+              or sign-ups required. has been the industry's standard dummy
+              text ever since the 1500s, when an unknown printer took a
+              standard dummy text.
+            </p>
+            <ul class="prog-langs">
+              <li>css</li>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <button type="button" class="btn">See Project</button>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="recent-work-container3">
+      <div class="my-works">
+        <div class="my-work-contain">
+          <div class="my-work-card">
+            <img src="./images/ImgPlaceholder.png" alt="image" />
+            <h3 class="title">Multy-Post Stories</h3>
+            <p class="work-description">
+              A daily selection of privately personalized reads; no accounts
+              or sign-ups required. has been the industry's standard dummy
+              text ever since the 1500s, when an unknown printer took a
+              standard dummy text.
+            </p>
+            <ul class="prog-langs">
+              <li>css</li>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
```

Comment:
- [ ] Kindly remove this horizontal line before CSS and use that CSS li at left side
### your design
![his](https://user-images.githubusercontent.com/81727988/178110191-72ab7c32-e726-4ffa-809e-aef8a265d1e6.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/178110202-7b063718-2915-4498-9452-41fb9c7268ed.png)